package.json

```diff
@@ -9,10 +9,14 @@
     "require": "./dist/cjs/index.cjs",
     "import": "./lib/index.js"
   },
+  "types": "types/index.d.ts",
   "browser": {
     "./lib/agentCompatibility.js": "./lib/agentCompatibility-browser.js",
     "./tests/utils.cjs": "./tests/utils-browser.cjs"
   },
+  "react-native": {
+    "./lib/agentCompatibility.js": "./lib/agentCompatibility-browser.js"
```

Comment:
Is this all that's needed?  Or is there an indirect need for the same sort of polyfill lib that the rdf-canonize patch had?  (https://github.com/digitalbazaar/rdf-canonize/pull/76)